### DISCUSSION TITLE
lore_process.onnx

### DISCUSSION THREAD

**Discussion by wangwenqi567** [SWHL/ConvertLOREToONNX#5] [Q&A]
lore_detect.onnx successfully converted.
运行main.py时报错，没有看到lore_process.onnx
RuntimeError: Expected all tensors to be on the same device, but found at least two devices, cuda:0 and cpu!
是都应该在cpu还是gpu呢，

**Comment by SWHL** [comment]
怀疑是你的环境下torch是GPU版的，请按照我提供的requirements重新创建一个环境来转

**Reply by wangwenqi567** [reply]
requirements的格式感觉有点奇怪，可以请教一下怎么直接安装requirements里面的包和库吗，谢谢，我用conda和pip都不太行

**Reply by SWHL** [reply]
我在README中写了呀：
https://github.com/SWHL/ConvertLOREToONNX#2-install-env

**Reply by wangwenqi567** [reply]
是的，我看到了 conda install --yes --file requirements.txt
但是我运行这个命令时报错，想向您再确认一下是这个命令吗，感谢回复

**Reply by SWHL** [reply]
参考最新README，我做了更新

**Reply by wangwenqi567** [reply]
好的，谢谢回复，我再尝试一下
